docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+| **Minimum distance to home (meters)**   | The minimum distance that the craft has to be from the home position to activate GPS Rescue. If the rescue starts too close to home, the craft will fly away until this distance is reached, then start the normal rescue behavior.                                                                                                                                                                                                               |
+| **Minimum satellites**                  | The minimum number of satellites required to activate GPS Rescue. If the number of satellites is below this value, GPS Rescue will not be activated.                                                                                                                                                                                                                                                                                              |
+| **Allow arming without fix**            | Allows arming without a GPS fix. If enabled and armed before GPS fix is acquired, the craft will disarm on loss of signal. If enabled, you cannot take off without a GPS fix.                                                                                                                                                                                                                                                                     |
+| **Sanity checks**                       | <li>**On (highly recommended)**: The craft will perform sanity checks before starting the rescue; if the checks fail, the craft will disarm.</li><li>**Off**: The craft will not perform sanity checks before starting the rescue, potentially leading to flying away or crashing.</li><li>**Failsafe only**: Sanity checks will only be performed on an actual loss of signal, not triggered if GPS Rescue is activated using the Aux mode.</li> |
+
+For more information about GPS Rescue on 4.5+, see this [Wiki document](/docs/wiki/guides/current/GPS-Rescue-v4-5).
+
+## Testing Failsafe
+
+:::danger
+It is very important to make sure your failsafe works as expected
+
+Failure to do so can result in lost or damaged equipment, or even injury to yourself or others.
+:::
+
+- When using the **Drop** failsafe, the easiest way to test it is to arm your craft **with your props off**,
+  then turn off your transmitter. The motors should stop.
+- When using other failsafe procedures, you can test them by settings up the **failsafe** mode on a switch,
+  and carefully activating it in an open area. If something goes wrong, you can revert the position of the switch
+  to stop the failsafe procedure.
+- For GPS Rescue, there is a **GPS Rescue** mode that can be activated independently of the failsafe mode.
```

Comment:
Add...
This mode will immediately activate Stage 2 GPS Rescue, bypassing Stage 1.